packages/datetime/test/datePickerTests.tsx

```diff
@@ -26,6 +30,139 @@ describe("<DatePicker>", () => {
         assert.isUndefined(root.state("selectedDay"));
     });
 
+    describe("reconciliates dayPickerProps", () => {
+        it("week starts with firstDayOfWeek value", () => {
+            const selectedFirstDay = 3;
+            const wrapper = mount(<DatePicker dayPickerProps={{ firstDayOfWeek: selectedFirstDay }} />);
+            const firstWeekday = wrapper.find("Weekday").first();
+            assert.equal(firstWeekday.prop("weekday"), selectedFirstDay);
+        });
+
+        it("doesnt show outside days if specified", () => {
+            const defaultValue = new Date(2017, Months.SEPTEMBER, 1, 12);
+            const wrapper = mount(
+                <DatePicker defaultValue={defaultValue} dayPickerProps={{ enableOutsideDays: false }} />,
+            );
+
+            const dayIsHidden = (day: any): boolean =>
+                day.prop("empty") && !day.prop("ariaSelected") && day.prop("ariaDisabled");
+
+            assert.isTrue(dayIsHidden(wrapper.find("Day").at(0)));
+            assert.isTrue(dayIsHidden(wrapper.find("Day").at(1)));
+            assert.isTrue(dayIsHidden(wrapper.find("Day").at(2)));
+            assert.isTrue(dayIsHidden(wrapper.find("Day").at(3)));
+            assert.isTrue(dayIsHidden(wrapper.find("Day").at(4)));
+
+            assert.isFalse(dayIsHidden(wrapper.find("Day").at(5)));
+        });
+
+        it("respects default of true for enableOutsideDays prop", () => {
```

Comment:
`shows outside days by default`

I'd put this test above the `doesn't show outside days if enableOutsideDays=false` test.